### PR TITLE
add message about log collection for PCI enabled orgs

### PR DESCRIPTION
**Pull request by alxch-**
### What does this PR do? What is the motivation?

We currently don't support PCI-enabled orgs for CI Visibility logs collection. This was not documented, though we need to let potential customers know. 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by alxch-** [comment]
I realize that I don't know how to make the footnotes work inside the `alert-info` boxes, this is not working properly. Please help on this.

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 1]
```diff
@@ -121,6 +121,8 @@ The AWS CodePipeline integration supports correlating **CodeBuild** actions with
 
 <div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility. Log retention, exclusion, and indexes are configured in Logs Settings. Logs for AWS CodeBuild can be identified by the <code>source:codebuild</code> and <code>sourcecategory:aws</code> tags.</div>
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for [PCI-compliant organizations][18].</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:124]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a></div>
```

Comment (michaelcretzman) [reply content/en/continuous_integration/pipelines/awscodepipeline.md:124]: You can't use Markdown in `<div>` tags.

File: content/en/continuous_integration/pipelines/azure.md [hunk 2]
```diff
@@ -102,6 +102,8 @@ Example for enabling the hooks in specified projects:
 
 <div class="alert alert-info">Azure Log Collection is in private beta. To request access, fill out <a href="https://forms.gle/vXEQQcPLARdSDLd27">this form</a>.</div>
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for [PCI-compliant organizations][15].</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/azure.md:105]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```

File: content/en/continuous_integration/pipelines/circleci.md [hunk 3]
```diff
@@ -89,6 +89,8 @@ To display and filter the teams associated with your pipelines, add `team:<your-
 
 The Datadog CircleCI integration collects logs from your finished CircleCI jobs and forwards them to Datadog.
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for [PCI-compliant organizations][16].</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/circleci.md:92]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```

File: content/en/continuous_integration/pipelines/github.md [hunk 4]
```diff
@@ -74,6 +74,8 @@ Pipelines appear immediately after enabling CI Visibility for any account or rep
 
 The GitHub Actions CI Visibility integration also allows automatically forwarding workflow job logs to [Datadog Log Management][5].
 
+<div class="alert alert-info"><strong>Note</strong>: Log collection is not available for [PCI-compliant organizations][17].</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/github.md:77]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 5]
```diff
@@ -265,6 +265,8 @@ The following GitLab versions support collecting job logs:
 
 <div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility.</div>
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for [PCI-compliant organizations][25].</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/gitlab.md:268]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```

**Review by michaelcretzman — APPROVED** [review]
Approving but I will add the edit suggestions.

**Comment by michaelcretzman** [comment]
@alxch- 
there are conflicts in this branch. Please resolve them. Thanks

**Comment by alxch-** [comment]
@michaelcretzman should be ready to re-review, thanks

File: content/en/continuous_integration/pipelines/azure.md [hunk 6]
```diff
@@ -102,6 +102,8 @@ Example for enabling the hooks in specified projects:
 
 <div class="alert alert-info">Azure Log Collection is in private beta. To request access, fill out <a href="https://forms.gle/vXEQQcPLARdSDLd27">this form</a>.</div>
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/azure.md:105]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations</a>.</div>
```

File: content/en/continuous_integration/pipelines/circleci.md [hunk 7]
```diff
@@ -89,6 +89,8 @@ To display and filter the teams associated with your pipelines, add `team:<your-
 
 The Datadog CircleCI integration collects logs from your finished CircleCI jobs and forwards them to Datadog.
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/circleci.md:92]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations</a>.</div>
```

File: content/en/continuous_integration/pipelines/github.md [hunk 8]
```diff
@@ -74,6 +74,8 @@ Pipelines appear immediately after enabling CI Visibility for any account or rep
 
 The GitHub Actions CI Visibility integration also allows automatically forwarding workflow job logs to [Datadog Log Management][5].
 
+<div class="alert alert-info"><strong>Note</strong>: Log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/github.md:77]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations</a>.</div>
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 9]
```diff
@@ -266,6 +266,8 @@ The following GitLab versions support collecting job logs:
 
 <div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility.</div>
 
+<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a>.</div>
```
Comment (michaelcretzman) [inline content/en/continuous_integration/pipelines/gitlab.md:269]:
```suggestion
<div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations</a>.</div>
```

**Review by michaelcretzman — APPROVED** [review]
I need to remove some double periods and then I'll merge.

**Pull request merged**